### PR TITLE
ci: migrate to Github Actions

### PR DESCRIPTION
**Pull request by erdemedeiros**
Part of https://github.com/Activiti/Activiti/issues/3898

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,9 +4,8 @@ repos:
     hooks:
       - id: helm-docs
   - repo: https://github.com/Alfresco/alfresco-build-tools
-    rev: v1.1.0
+    rev: v1.3.0
```
Comment (gicappa) [inline .pre-commit-config.yaml:7]: isn't the latest `v.1.4.0` ?

Comment (erdemedeiros) [reply .pre-commit-config.yaml]: Good catch! Fixed.

**Review by andrea-ligios — APPROVED** [review]
LGTM!

**Review by gicappa — APPROVED** [review]
LGTM!

**Pull request merged**